OpenTrons/opentrons-api
# feat(app): add desktop and ODD droptip wizard loading/confirmation/errors on app

closes [RAUT-796](https://opentrons.atlassian.net/browse/RAUT-796)

# Overview

Adds functionality to Droptip Wizard, both on desktop and ODD. Handles loading, error, and confirmation states for robot gantry movement, blowout action, and tip drop action. Adds video assets for initial blowout/tip drop selection.

# Test Plan

- Verify selections proceed properly for blowout/tip drop selecitons
- Verify deck slot selection and jog controls work properly
- Verify loading states correspond to action that is being performed
- Verify confirmation tile renders after jogging to ensure the intended position is reached before blowout/tip drop

* Note that ODD testing is contingent on [this branch](https://github.com/Opentrons/opentrons/compare/edge...ODD_ENTRY_POINT_DROP_TIP) being merged to expose ODD entrypoint from instrument detail page

# Review requests

@mjhuff (collaborator)

# Risk assessment

medium

[RAUT-796]: https://opentrons.atlassian.net/browse/RAUT-796?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
